src/helpers/dateTime.ts

```diff
@@ -0,0 +1,30 @@
+import moment from 'moment'
+const months = {
+  0: "Jan",
```

Comment:
if we are importing moment, why don't we use it to do this work for us?

something like..

```
    if date.getFullYear() === now.getFullYear() {
        return moment(date).format("MMM DD");
    } else {
        return moment(date).format("MMM DD YYYY");
    }
```

Reply:
In our requirement, we don't need that format, instead, "a few seconds ago", "a month ago", etc. Am I correct?